test/TestProjects/MgmtScopeResource/Generated/Extensions/ArmResourceExtensions.cs

```diff
@@ -12,14 +12,11 @@ namespace MgmtScopeResource
     /// <summary> A class to add extension methods to ArmResource. </summary>
     public static partial class ArmResourceExtensions
     {
-        /// <summary> Gets a collection of PolicyAssignments in the PolicyAssignment. </summary>
-        /// <param name="armResource"> The <see cref="ArmResource" /> instance the method will execute against. </param>
-        /// <returns> An object representing collection of PolicyAssignments and their operations over a PolicyAssignment. </returns>
-        public static PolicyAssignmentCollection GetPolicyAssignments(this ArmResource armResource)
```

Comment:
What happened to the GetPolicyAssignments method?

Reply:
As discussed, changed name to FakePolicyAssignments